__tests__/app.test.js

```diff
@@ -84,4 +84,27 @@ describe('GET /api/articles', () => {
                 expect(body.msg).toBe('No article found for article_id: 14')
             })
     })
+})
+describe('GET /api/articles', () => {
+    test('GET 200 returns an array of article objects with the expected properties', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body
+                expect(articles.length).toBe(13)
+                expect(articles).toBeSortedBy('created_at', { descending: true })
+                articles.forEach((article) => {
+                    expect(typeof article.article_id).toBe('number')
+                    expect(typeof article.author).toBe('string')
+                    expect(typeof article.title).toBe('string')
+                    expect(typeof article.topic).toBe('string')
+                    expect(typeof article.created_at).toBe('string')
+                    expect(typeof article.votes).toBe('number')
+                    expect(typeof article.article_img_url).toBe('string')
+                    expect(typeof article.comment_count).toBe('number')
+                })
+                expect(articles.map((article) => article.comment_count)).toEqual([2, 1, 0, 0, 0, 2, 11, 2, 0, 0, 0, 0, 0])
+            })
```

Comment:
Great test, well done for 
- checking the length of the array to avoid false positives
- keeping the test flexible by checking the object contains certain keys and the datatypes of the keys

You could improve this by
- adding a test to check the returned articles do not contain the body key, if they did this currently this test would still pass
- have a look at using matchers like `.toMatchObject()` to save you some typing!